tests/Behat/Context/Ui/Admin/ManagingProductsContext.php

```diff
@@ -65,4 +75,16 @@ public function iShouldBeNotifiedThatItHasBeenAlreadyEnqueued()
             $this->notificationChecker, NotificationType::success(), 'Akeneo PIM import for this product has been already scheduled before'
         );
     }
+
+    /**
+     * @Then /^I should see (\d+), not imported, items? in the Akeneo queue items list$/
+     */
+    public function iShouldSeeNotImportedItemInTheAkeneoQueueItemsList(int $numberOfItems)
```

Comment:
Is there any reason why we put these methods in the ManagingProductContext class?